### PR TITLE
adds attributes to multiple serializers

### PR DESCRIPTION
**Pull request by george-silva**
1. adds area_acres to treatment prescription [list] serializer
2. adds uuid and id to treatment plan serializer

### CODE REVIEW

**Review by pflopez — APPROVED** [review]

File: src/planscape/impacts/serializers.py [hunk 1]
```diff
@@ -100,14 +118,17 @@ class TreamentPrescriptionUpsertSerializer(serializers.Serializer):
...
         model = TreatmentPrescription
         fields = (
+            "id",
             "uuid",
             "created_at",
             "created_by",
             "updated_at",
+            "project_area",
             "action",
             "stand",
```
Comment (pflopez) [inline src/planscape/impacts/serializers.py:132]: not part of this pr, but should this be `stands` ?

**Review by lastminutediorama — APPROVED** [review]
